lib/recipe.rb

```diff
@@ -45,7 +45,8 @@
 agent_bin         = ChefConfig::PathHelper.join(agent_bin_dir, 'automate-liveness-agent')
 agent_conf        = ChefConfig::PathHelper.join(agent_etc_dir, 'config.json')
 agent_username    = 'chefautomate'
-run_interval      = 1 # only windows
+# TODO: This needs to be 30 minutes, except when we're running tests
+run_interval      = 1 # (only windows)
```

Comment:
Since this needs to be 30 minutes shouldn't we set it to 30 minutes before we ship it?